Source/CesiumRuntime/Private/CesiumRuntime.cpp

```diff
@@ -90,12 +91,17 @@ std::string getCacheDatabaseName() {
 } // namespace
 
 const std::shared_ptr<CesiumAsync::IAssetAccessor>& getAssetAccessor() {
+  int RequestsPerCachePrune =
+      GetDefault<UCesiumRuntimeSettings>()->RequestsPerCachePrune;
+  int MaxCacheItems = GetDefault<UCesiumRuntimeSettings>()->MaxCacheItems;
```

Comment:
This should be static so that they're not execute unnecessarily on every invocation of this function. You could also put them inline in the static assignment below, if you prefer.